master/internal/api_command.go

```diff
@@ -103,41 +99,16 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 	if req.MustZeroSlot {
 		resources.Slots = 0
 	}
-	poolName, err := a.m.rm.ResolveResourcePool(
-		resources.ResourcePool, int(cmdSpec.Metadata.WorkspaceID), resources.Slots)
+	poolName, launchWarnings, err := ResolveResources(a, resources.ResourcePool, resources.Slots, int(cmdSpec.Metadata.WorkspaceID))
```

Comment:
great, is it possible to reuse these for the existing experiments in `internal/core_experiment.go` and/or `internal/experiment.go`?

Reply:
I'd love if this became the RM api and all the RM apis it calls just became internal.

Reply:
added to core_experiment.go